linux/installation/mde_installer.sh

```diff
@@ -984,6 +985,22 @@ set_epp_to_passive_mode()
     log_info "[v] passive mode set"
 }
 
+set_epp_to_rtp_mode()
```

Comment:
you can have one common function (combine set_epp_to_passive_mode. and this one)

Reply:
There is no duplication of code except for first 3 lines.